src/api_server/Cargo.toml

```diff
@@ -11,6 +11,7 @@ serde = { version = ">=1.0.27", features = ["derive"] }
 serde_derive = ">=1.0.27"
 serde_json = ">=1.0.9"
 derive_more = { version = "0.99.17", default-features = false, features = ["from"] }
+thiserror = "1.0.32"
```

Comment:
> Is there any advantage to thiserror other than convenience?

No, but it is a large convenience, across Firecracker it would remove hundreds of largely identical lines of code.